### PR TITLE
infra: Make docker folder read only, remove unused tests, update repo name, remove incorrect documentation.

### PR DESCRIPTION
**Pull request by nadiaya**
* Make docker folder read only
* Remove unused tests
* Rename test-toolkit/ -> test/
* Rename buildspec.yml -> buildspec-container-pr.yml 
* Rename buildspec-toolkit.yml -> buildspec.yml 
* Update repo name
* Remove incorrect documentation.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ GitHub provides additional document on [forking a repository](https://help.githu
 
 
 ## Finding contributions to work on
-Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels (enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/aws/sagemaker-mxnet-serving-container/labels/help%20wanted) issues is a great place to start.
+Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels (enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/aws/sagemaker-mxnet-inference-toolkit/labels/help%20wanted) issues is a great place to start.
```
Comment (laurenyu) [inline CONTRIBUTING.md:44]: our issue labels are actually slightly different than the defaults - most relevant is that we've actually deleted the "help wanted" label that's specifically linked here 😂

File: test/conftest.py [hunk 2]
```diff
@@ -20,123 +20,155 @@
...
-from test.integration import NO_P2_REGIONS, NO_P3_REGIONS
+from utils import image_utils
 
 logger = logging.getLogger(__name__)
-logging.getLogger("boto").setLevel(logging.INFO)
-logging.getLogger("botocore").setLevel(logging.INFO)
-logging.getLogger("factory.py").setLevel(logging.INFO)
-logging.getLogger("auth.py").setLevel(logging.INFO)
-logging.getLogger("connectionpool.py").setLevel(logging.INFO)
+logging.getLogger('boto').setLevel(logging.INFO)
```
Comment (laurenyu) [inline test/conftest.py:26]: if we use black to format this repo too, we're going to end up changing these back to double quotes 🤷‍♀️

File: test/conftest.py [hunk 3]
```diff
@@ -20,123 +20,155 @@
...
+    parser.addoption('--aws-id', default=None)
+    parser.addoption('--instance-type', default=None)
+    parser.addoption('--accelerator-type', default=None)
     # If not specified, will default to {framework-version}-{processor}-py{py-version}
-    parser.addoption("--tag", default=None)
+    parser.addoption('--tag', default=None)
+
+
+@pytest.fixture(scope='session', name='dockerfile_type')
+def fixture_dockerfile_type(request):
```
Comment (laurenyu) [inline test/conftest.py:62]: nit: the other fixtures in this file are defined without the `fixture_` prefix

File: tox.ini [hunk 4]
```diff
@@ -13,9 +13,9 @@ exclude =
     __pycache__
     .tox
     test/resources
-    test-toolkit/resources
     venv/
     docker/
+    artifacts/
```
Comment (laurenyu) [inline tox.ini:18]: there is a Python file in there - does it make sense to exclude?

File: CONTRIBUTING.md (hunk 1 above)
Comment (nadiaya) [reply CONTRIBUTING.md:44]: This is default contributing file and we should probably change them on all repo to be the same  🤷‍♀️

File: test/conftest.py (hunk 2 above)
Comment (nadiaya) [reply test/conftest.py:26]: Git weird - I moved this file and didn't really change that.

Unfortunately we use both single  and double quotes all over the repo :(

File: tox.ini (hunk 4 above)
Comment (nadiaya) [reply tox.ini:18]: It was excluded before🤷‍♀️

**Pull request merged**